contributor-docs/documentation.md

```diff
@@ -0,0 +1,9 @@
+# Documentation
+
+## YARD setup
+
+## Writing Storybook stories
```

Comment:
Should this be here or in 'Component tests'? I think Storybook is most useful as a development tool rather than as documentation, but it does also act as documentation 🤔